mobile-app/app/tailwind.config.js

```diff
@@ -117,6 +121,11 @@ module.exports = {
       },
       zIndex: {
         '-1': '-1'
+      },
+      fontSize: {
```

Comment:
I prefer not introducing a new font for this. Just use text-xs for the "Active" tag. and maybe increase the size of Address text